target/linux/kirkwood/modules.mk

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
```

Comment:
This should be updated to 2020.

Personally, I'd remove it from such a trivial file, though.